daisy_workflows/image_build/enterprise_linux/export_metadata.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Export image metadata on a GCE VM.
+
+Parameters (retrieved from instance metadata):
+google_cloud_repo: The package repo to use. Can be stable (default), staging,
+  or unstable.
+el_release: rhel6, rhel7, centos6, centos7, oraclelinux6, or oraclelinux7
+image_dest: The Cloud Storage destination for the resultant image.
+destination_prefix: The CS path to export image metadata to
+"""
+
+import json
+import logging
+import os
+import time
+import uuid
+
+import utils
+
+
+def main():
+  # Get Parameters.
+  repo = utils.GetMetadataAttribute('google_cloud_repo', raise_on_not_found=True).strip()
+  el_release_name = utils.GetMetadataAttribute('el_release', raise_on_not_found=True)
+  build_date = utils.GetMetadataAttribute('build_date', raise_on_not_found=True)
+  destination_prefix = utils.GetMetadataAttribute('destination_prefix', raise_on_not_found=True)
+  uefi = utils.GetMetadataAttribute('rhel_uefi') == 'true'
+
+  # Mount the installer disk.
+  if uefi:
+    utils.Execute(['mount', '/dev/sdb2', '/mnt'])
+  else:
+    utils.Execute(['mount', '/dev/sdb1', '/mnt'])
+
+  logging.info('Installer root: %s' % os.listdir('/mnt'))
+
+  # Create and upload metadata of the image and packages
+  logging.info('Creating image metadata.')
+  release_time = time.time()
+  image = {
+      "id": uuid.uuid4(),
+      "family": el_release_name,
+      "name": el_release_name + build_date,
+      "version": build_date,
+      "location": "gs://gce-image-archive/centos-uefi/centos-7-v${build_date}",
+      "release_date": build_date,
+      "release_time": release_time,
+      "state": "Active",
```

Comment:
is this meant to be active/deprecated? i think that you'll have to pull from GCE API, rather than here